snapcraft/internal/elf.py

```diff
@@ -619,6 +623,10 @@ def get_elf_files(root: str, file_list: Sequence[str]) -> FrozenSet[ElfFile]:
         except elftools.common.exceptions.ELFError:
             # Ignore invalid ELF files.
             continue
+        except errors.CorruptedElfFile as e:
```

Comment:
we are trying to move away from "e", can you use a more descriptive var name?

Reply:
Sure, done.